Tools/limine/limine.cfg

```diff
@@ -0,0 +1,6 @@
+TIMEOUT=5
```

Comment:
Future. Let's create the config file within Starter.cs so it can be customized later.